app/uk/gov/hmrc/securemessage/services/SecureMessageService.scala

```diff
@@ -66,17 +66,27 @@ class SecureMessageService @Inject()(
     } yield ()
   }.value
 
-  def getConversationsFiltered(authEnrolments: Enrolments, filters: ConversationFilters)(
+  def getConversations(authEnrolments: Enrolments, filters: Filters)(
     implicit ec: ExecutionContext,
     messages: Messages): Future[List[ConversationMetadata]] = {
     val filteredEnrolments = authEnrolments.filter(filters.enrolmentKeysFilter, filters.enrolmentsFilter)
     val identifiers: Set[Identifier] = filteredEnrolments.map(_.asIdentifier)
-    conversationRepository.getConversationsFiltered(identifiers, filters.tags).map {
+    conversationRepository.getConversations(identifiers, filters.tags).map {
       _.map(ConversationMetadata.coreToConversationMetadata(_, identifiers)) //TODO: move this to controllers
         .sortBy(_.issueDate.getMillis)(Ordering[Long].reverse)
     }
   }
 
+  def getMessages(authEnrolments: Enrolments, filters: Filters)(
+    implicit ec: ExecutionContext): Future[List[Message]] = {
+    val filteredEnrolments = authEnrolments.filter(filters.enrolmentKeysFilter, filters.enrolmentsFilter)
+    val identifiers: Set[Identifier] = filteredEnrolments.map(_.asIdentifier)
+    for {
+      conversations <- conversationRepository.getConversations(identifiers, filters.tags)
+      letters       <- messageRepository.getLetters(identifiers, filters.tags)
+    } yield conversations ++ letters
```

Comment:
do we not want the result ordered?

Reply:
hmm, let me see what we do on conversation metadata...

Reply:
let's hope we don't introduce paging soon because sorting for a paged resultset will be an interesting problem to solve